### PR TITLE
Add Code of Conduct

### PR DESCRIPTION
**Pull request by nicklathe**
- Derived from https://www.contributor-covenant.org

### CODE REVIEW

File: CODE-OF-CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+when an individual is representing Scipian or its community. Examples of
+representing a project or community include using an official Scipian e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of Scipian may be
+further defined and clarified by Scpian project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the Scipian project team at TODO(NAL):[INSERT EMAIL ADDRESS].
```
Comment (ncronquist) [inline CODE-OF-CONDUCT.md]: Do we need to create a ServiceNow ticket to get this email alias created?

Comment (dragan) [reply CODE-OF-CONDUCT.md]: Not sure, but I think to begin, we should just use my email address for now.

Comment (ncronquist) [reply CODE-OF-CONDUCT.md]: Agreed. I think we should have an email address added before committing this. We should also create an issue to follow up on creating a Scipian email alias.

Comment (dragan) [reply CODE-OF-CONDUCT.md]: Yes, but it should be on the `scipian.io` domain or maybe a hosted mailing list. No need to get into details here, let's create another issue to look into public mailing lists and etc.

Comment (nicklathe) [reply CODE-OF-CONDUCT.md]: @dragan so for now, you want me to put your SAP email here?

Comment (dragan) [reply CODE-OF-CONDUCT.md]: Yes

**Review by dragan — CHANGES_REQUESTED** [review]
LGTM, squash the commits.

**Review by dragan — APPROVED** [review]
:shipit:

Ship It

**Review by ncronquist — APPROVED** [review]
:shipit: 

thank you

**Comment by ncronquist** [comment]
Created https://github.com/scipian/community/issues/16 to track the creation of an email alias for the Scipian project maintainers.

**Pull request merged**